test/react-native-cli/features/steps/steps.rb

```diff
@@ -210,9 +212,16 @@ def parse_package_json
   step("the interactive file '#{filename}' contains '[Bugsnag start];'")
 end
 
-Then("the Android app contains the bugsnag initialisation code") do
-  filename = "android/app/src/main/java/com/#{current_fixture}/MainApplication.java"
+def get_android_main_application_path(current_fixture)
+  if current_fixture.include? 'expo_ejected'
+     "android/app/src/main/java/com/bugsnag/#{current_fixture}/MainApplication.java"
+  else
+    "android/app/src/main/java/com/#{current_fixture}/MainApplication.java"
+  end
```

Comment:
There's an extra space here 😱

```suggestion
  if current_fixture.include? 'expo_ejected'
    "android/app/src/main/java/com/bugsnag/#{current_fixture}/MainApplication.java"
  else
    "android/app/src/main/java/com/#{current_fixture}/MainApplication.java"
  end
```